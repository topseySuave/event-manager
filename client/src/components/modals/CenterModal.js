import React, { Component } from 'react';
import AddCenterForm from './modalForms/addCenterForm'

class CenterModal extends Component {

    render() {
        return (
            <div id="edit_center_modal" className="modal modal-fixed-footer">
                <div className="modal-content">
                    <h4><span>Create</span> Center</h4>
                    <div className="row">
                        <AddCenterForm />
                    </div>
                </div>
                <div className="modal-footer">
                    <a className="modal-action modal-close waves-effect btn-flat">Cancel</a>
                    {/*<a onClick={this.onClick()} className="btn waves-effect gradient__bg lighten-1">Create</a>*/}
                </div>
            </div>
        );
    }
}

export default CenterModal;
