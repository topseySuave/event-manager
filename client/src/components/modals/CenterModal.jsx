import React, { Component } from 'react';
import AddCenterForm from './centerModalForms/addCenterForm';

/**
 * CenterModal Class Component
 * */
class CenterModal extends Component {
  /**
   * render Method Component
   * @return { component }
   * */
  render() {
    return (
      <div id="edit_center_modal" className="modal modal-fixed-footer">
        <div className="modal-content">
          <h4>
            <span>Create</span> Center
          </h4>
          <div className="row">
            <AddCenterForm />
          </div>
        </div>
        <div className="modal-footer">
          <a className="modal-action modal-close waves-effect btn-flat">
            Cancel
          </a>
        </div>
      </div>
    );
  }
}

export default CenterModal;
