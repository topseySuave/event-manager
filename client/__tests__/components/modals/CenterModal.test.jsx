import React from 'react';
import { mount, shallow } from 'enzyme';
import CenterModal from
  '../../../src/components/modals/CenterModal';

describe('CenterModal component', () => {
  const wrapper = shallow(<CenterModal />);
  test('take a snapshot of the mounted component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
