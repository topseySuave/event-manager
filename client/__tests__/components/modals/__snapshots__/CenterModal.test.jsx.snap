// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CenterModal component take a snapshot of the mounted component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CenterModal />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="modal-content"
        >
          <h4>
            <span>
              Create
            </span>
             Center
          </h4>
          <div
            className="row"
          >
            <Connect(AddCenterForm) />
          </div>
        </div>,
        <div
          className="modal-footer"
        >
          <a
            className="modal-action modal-close waves-effect btn-flat"
          >
            Cancel
          </a>
        </div>,
      ],
      "className": "modal modal-fixed-footer",
      "id": "edit_center_modal",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h4>
              <span>
                Create
              </span>
               Center
            </h4>,
            <div
              className="row"
            >
              <Connect(AddCenterForm) />
            </div>,
          ],
          "className": "modal-content",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <span>
                  Create
                </span>,
                " Center",
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Create",
                },
                "ref": null,
                "rendered": "Create",
                "type": "span",
              },
              " Center",
            ],
            "type": "h4",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Connect(AddCenterForm) />,
              "className": "row",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <a
            className="modal-action modal-close waves-effect btn-flat"
          >
            Cancel
          </a>,
          "className": "modal-footer",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Cancel",
            "className": "modal-action modal-close waves-effect btn-flat",
          },
          "ref": null,
          "rendered": "Cancel",
          "type": "a",
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="modal-content"
          >
            <h4>
              <span>
                Create
              </span>
               Center
            </h4>
            <div
              className="row"
            >
              <Connect(AddCenterForm) />
            </div>
          </div>,
          <div
            className="modal-footer"
          >
            <a
              className="modal-action modal-close waves-effect btn-flat"
            >
              Cancel
            </a>
          </div>,
        ],
        "className": "modal modal-fixed-footer",
        "id": "edit_center_modal",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h4>
                <span>
                  Create
                </span>
                 Center
              </h4>,
              <div
                className="row"
              >
                <Connect(AddCenterForm) />
              </div>,
            ],
            "className": "modal-content",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <span>
                    Create
                  </span>,
                  " Center",
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Create",
                  },
                  "ref": null,
                  "rendered": "Create",
                  "type": "span",
                },
                " Center",
              ],
              "type": "h4",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Connect(AddCenterForm) />,
                "className": "row",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <a
              className="modal-action modal-close waves-effect btn-flat"
            >
              Cancel
            </a>,
            "className": "modal-footer",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Cancel",
              "className": "modal-action modal-close waves-effect btn-flat",
            },
            "ref": null,
            "rendered": "Cancel",
            "type": "a",
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
